test/dummy/db/migrate/20210425231529_create_spina_admin_journal_journals.spina_admin_journal_engine.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+# This migration comes from spina_admin_journal_engine (originally 20201216152147)
+
+class CreateSpinaAdminJournalJournals < ActiveRecord::Migration[6.0] # :nodoc:
+  def change
+    create_table :spina_admin_journal_journals do |t|
+      t.string :name, null: false, default: 'Unnamed Journal'
+      t.integer :singleton_guard, null: false, index: { unique: true }
```

Comment:
Is this a custom datatype?

Reply:
It serves only to prevent multiple journals ever being created (unless a future patch enables this).